src/app/navbar/navbar.component.html

```diff
@@ -57,6 +57,17 @@
         </div>
       </li>
 
+      <li class="nav-item" ngbDropdown>
+        <a class="nav-link dropdown-toggle" id="categoryDropdown" role="button"
+           aria-haspopup="true" aria-expanded="false" ngbDropdownToggle>Categorías</a>
```

Comment:
Please, to keep a coherent UI use English (Categories...). Also consider adding the Create menu option, which should be just shown to admins...